### PR TITLE
database contains 4 tables with relations

### LINKED ISSUE #138: Scripts for initializing databases
Generate databases for development, production and testing.

### PR DESCRIPTION
**Pull request by att78**
closes #138 

Updates database to match current plan for database structure. Tech design is available at google drive

### CODE REVIEW

**Review by ilkkamaksy — APPROVED** [review]
lgtm

**Comment by guotin** [comment]
Now that #142 showed that we might need a login/registering with our own way, should some password field be added to `USER` already? @ilkkamaksy ?

File: userdata/init_scripts/init.sql [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+    service_id INTEGER REFERENCES SERVICES (service_id),
+    username VARCHAR, 
+    email VARCHAR,
+    token VARCHAR,
+    reposurl VARCHAR
+);
+
+CREATE TABLE REPO(
+    repo_id serial PRIMARY KEY,
+    service_user_id INTEGER REFERENCES SERVICE_USERS (service_user_id),
```
[the remark this answers is not in the document]
Comment (att78) [reply userdata/init_scripts/init.sql:26]: Good idea. Ill change it

**Comment by att78** [comment]
I can add the password field as well.

**Review by guotin — APPROVED** [review]

**Pull request merged**